aws/resource_aws_route53_health_check_test.go

```diff
@@ -279,6 +279,7 @@ resource "aws_route53_health_check" "foo" {
   request_interval = "30"
   measure_latency = true
   invert_healthcheck = true
+  disabled = false
```

Comment:
To ensure we are not introducing any regressions (unlikely given the implementation here, but as a good practice), could we leave the existing test configurations without these changes and introduce a new acceptance test with its own configuration that verifies the new functionality? e.g.

```go
func TestAccAWSRoute53HealthCheck_Disabled(t *testing.T) {
  resourceName := "aws_route53_health_check.test"

  resource.ParallelTest(t, resource.TestCase{
    PreCheck:      func() { testAccPreCheck(t) },
    Providers:     testAccProviders,
    CheckDestroy:  testAccCheckRoute53HealthCheckDestroy,
    Steps: []resource.TestStep{
      {
        Config: testAccRoute53HealthCheckConfigDisabled(true),
        Check: resource.ComposeTestCheckFunc(
          testAccCheckRoute53HealthCheckExists(resourceName),
          resource.TestCheckResourceAttr(resourceName, "disabled", "true"),
        ),
      },
      {
        ResourceName:      resourceName,
        ImportState:       true,
        ImportStateVerify: true,
      },
      {
        Config: testAccRoute53HealthCheckConfigDisabled(false),
        Check: resource.ComposeTestCheckFunc(
          testAccCheckRoute53HealthCheckExists(resourceName),
          resource.TestCheckResourceAttr(resourceName, "disabled", "false"),
        ),
      },
      {
        Config: testAccRoute53HealthCheckConfigDisabled(true),
        Check: resource.ComposeTestCheckFunc(
          testAccCheckRoute53HealthCheckExists(resourceName),
          resource.TestCheckResourceAttr(resourceName, "disabled", "true"),
        ),
      },
    },
  })
}

func testAccRoute53HealthCheckConfigDisabled(disabled bool) string {
  return fmt.Sprintf(`
resource "aws_route53_health_check" "test" {
  disabled          = %[1]t
  failure_threshold = "2"
  fqdn              = "dev.notexample.com"
  port              = 80
  request_interval  = "30"
  resource_path     = "/"
  type              = "HTTP"
}
`, disabled)
```